lib/internal/util.js

```diff
@@ -500,6 +501,8 @@ const lazyDOMException = hideStackFrames((message, name) => {
 const kEnumerableProperty = ObjectCreate(null);
 kEnumerableProperty.enumerable = true;
```

Comment:
Should we freeze `kEnumerableProperty` as well?

Reply:
Not in this PR; but yes, it would make sense.